DataFormats/Detectors/MUON/MCH/include/DataFormatsMCH/DsChannelGroup.h

```diff
@@ -0,0 +1,63 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_MCH_DSCHANNELGROUP_H_
+#define ALICEO2_MCH_DSCHANNELGROUP_H_
+
+#include <vector>
+#include "Rtypes.h"
+
+namespace o2
+{
+namespace mch
+{
+
+class DsChannelId
```

Comment:
that might not have to be solved for this PR, but ... we actually have two ways to reference the channels : (solarId,dsId,channel) (raw/online data oriented) and (deId,dsId,channel) (digit/offline oriented) and both can be encoded to 32 bits. 

I was wondering it we could not decide to call those channelIds (== uint32_t) and use a couple of bits in those 32 to indicate which convention is used (solar or de) ?

Reply:
Indeed we could use the most significant bit to indicate the type of encoding, and two different constructors (`DsChannelId(uint16_t solarId, uint8_t dsId, uint8_t channel)` or `DsChannelId(uint16_t deId, uint16_t padId)`) for the two cases.

As a starting point, I will add getter functions for the solarId/dsId/channel values, to avoid the direct manipulation of the 32-bit encoded value.